plugins/services/src/js/filters/TaskNameTextFilter.js

```diff
@@ -23,7 +23,10 @@ class TaskNameTextFilter extends DSLFilter {
    */
   filterApply(resultSet, filterType, filterArguments) {
     return resultSet.filterItems(task => {
-      return task.name.indexOf(filterArguments.text) !== -1;
+      return (
+        (task.id && task.id.indexOf(filterArguments.text) !== -1) ||
```

Comment:
not blocking, but I'd love us using `includes` instead of `indexOf` in these cases 👍  (same below with `task.name.indexOf`)